desktop/src/main/java/bisq/desktop/main/portfolio/duplicateoffer/DuplicateOfferView.java

```diff
@@ -61,7 +61,7 @@ protected void doActivate() {
         onPaymentAccountsComboBoxSelected();
     }
 
-    public void initWithData(OfferPayload offerPayload) {
+    public void initWithData(OfferPayloadBase offerPayload) {
```

Comment:
Is it required to change the type to OfferPayloadBase as it is only used by non-BSQ-Swap offers.

Reply:
No can stay with the old Type.